src/fsdocs-tool/Options.fs

```diff
@@ -38,3 +38,131 @@ module Common =
         if b then
             printf "\nPress any key to continue ..."
             System.Console.ReadKey() |> ignore
+
+    open System.IO
+
+    /// <summary>
+    /// a set of default locations in this repo.
+    /// these files are to be used in 2 scenarios assuming we are executing directly from
+    ///
+    ///   src\fsdocs-tool\bin\Debug\net6.0\fsdocs.exe
+    ///
+    ///   src\fsdocs-tool\bin\Release\net6.0\fsdocs.exe:
+    ///
+    /// - as default styles when running watch or build when there are no user equivalents present and `nodefaultcontent` is not set to true
+    ///
+    /// - as content of the output of the `init` command to initialize a default docs folder structure.
+    ///
+    /// Note that the path of these files will always be combined with the given `assemblyPath` because the cli tool will query it's own path on runtime via reflection.
+    /// </summary>
+    type InRepoLocations(relAssemblyPath) =
```

Comment:
Here i am not sure, because i use the same class to determine the destination paths of the files to write, so the paths are actually correct for the "target repo" as well as the local FSharp.Formatting repo